.changeset/wet-foxes-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-typescript": patch
+---
+
+feat: add dedupeEnums option
```

Comment:
Since we’re adding a feature, this could be a minor bump.

For openapi-fetch, we are still < 1.0 so the logic is different: breaking changes for minor; patch everything else. But this package follows proper semver. But not a big deal either way.